train.py

```diff
@@ -706,7 +708,13 @@ def main():
 def train_one_epoch(
         epoch, model, loader, optimizer, loss_fn, args,
         lr_scheduler=None, saver=None, output_dir=None, amp_autocast=suppress,
-        loss_scaler=None, model_ema=None, mixup_fn=None, cuda=True):
+        loss_scaler=None, model_ema=None, mixup_fn=None, cuda=True, qat = False):
+
+    if qat:
```

Comment:
can we move this to the base train function so we only need to run this "disabling half precision..." step once?